Assets/Scripts/Enemy.cs

```diff
@@ -4,12 +4,27 @@
 
 public class Enemy : Entity
 {
+    //the target position of this enemy
+    protected Transform target;
+
     private void Start()
     {
-        MaxHealth = 1;
+        MaxHealth = 5;
         Health = MaxHealth;
         Strength = 1;
-        Speed = 20;
+        Speed = 3;
+
+        //locate Player, let this be the target
+        target = GameObject.FindGameObjectWithTag("Player").transform;
```

Comment:
I'm wondering, what if the player is an adjacent room? Wouldn't that mean the enemies would run into walls trying to get to the player?

Reply:
Well as all enemies must be dead before a room is exited, we need not worry here.